lib/class-wp-theme-json-gutenberg.php

```diff
@@ -956,6 +956,27 @@ public function get_settings() {
 		}
 	}
 
+	/**
+	 * Processes the CSS, to apply nesting.
+	 *
+	 * @param string $css      The CSS to process.
+	 * @param string $selector The selector to nest.
+	 *
+	 * @return string The processed CSS.
+	 */
+	public function process_blocks_custom_css( $css, $selector ) {
```

Comment:
Are we sure we want this method to be public?

Reply:
Well... In the beginning, I made it public to make testing the method easier. 
But the more I think about it, the more I like it... The method doesn't change anything in the object itself, so I don't see a reason to make it private, as it's pretty self-isolated. This way it will be possible for 3rd parties to use it in their own implementations if they choose to do something with nested CSS. In fact, we could even make it a static function and nothing would change. Perhaps it would even be preferable? 🤷

Reply:
That's the reason I asked. If the method is `public` just for unit tests, then we can make it `protected` and test the expected output using `get_stylesheet()`. We already do this for global custom CSS. 

If you think it could be helpful outside of this particular case, then sure, keeping it public makes sense.